metrics-jersey/pom.xml

```diff
@@ -23,6 +23,10 @@
       <artifactId>micrometer-core</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
+      <artifactId>dropwizard-configuration</artifactId>
```

Comment:
Needed just for type `ConfigurationSourceProvider` that we implement and register to allow configuration resource/file overrides.